client/src/App.vue

```diff
@@ -230,8 +230,6 @@ export default class App extends Mixins(SessionTimerMixin) {
 #main-column
   padding: 0
   margin: 0 auto 0 auto
-  width: 1380px
```

Comment:
Are there any responsive breakpoints to update?

Reply:
Not that i could see..